_shows/18_19/2_clowns_1_cup.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 2 Clowns 1 Cup
+company: Ugly Bucket Theatre
+period: Spring
+season: StuFF
+season_sort: 570
+date_start: 2019-06-18
+date_end: 2019-06-18
+venue: New Theatre Studio A
+
+cast:
+- name: Grace Gallagher
+  person: false
+- name: Crystal Quinney Barella
+  person: false
+
+crew_incomplete: true 
```

Comment:
did they have more crew??

Reply:
I suspect there was a producer, stage manager and sound designer, but unknown who these people are.